### DISCUSSION TITLE
Building a bootstrapper bundle as a stand alone project

### DISCUSSION THREAD

**Discussion by SamOxigen** [wixtoolset/issues#6900] [Questions]
- Wix has a built-in variable spelled as "var._ProjectName_.Configuration" that holds a build-in configuration name from the project (Debug/Release etc.).
- We build the wix project with Python as a separate task; Building the solution does not build the wix (bootstrapper project). So, we build the solution first -that builds the WPF project, then we build the wix project right after. This builds the bootstrapper bundle. 
- There is no build issues whatsoever when building the solution from within the Visual Studio IDE, either the solution or all projects separately.
- However, when building the wix project from the command line alone -this, "var._ProjectName_.Configuration" variable is not getting resolved for some reason.
- It did work however using this, "var.Configuration" variable, even though there is no such documented variable on wix web site (https://wixtoolset.org/documentation/manual/v3/votive/votive_project_references.html). As a matter of fact 😉, It did work in both, on Visual Studio IDE and on command line calls!

Any thought why "var.<(bootstrapper)_ProjectName_>.Configuration" variable is not resolved, and how come this, "var.Configuration" variable works?

**Comment by robmen** [comment]
Some define constants come from the project and some define constants come from the solution. Add an MSBuid Property `<DefineSolutionProperties>false</DefineSolutionProperties>` and you'll never accidentally use solution constants. Or, always build your .sln file.
